### PR TITLE
Added Json Validator support #1367

### PR DESCRIPTION
**Pull request by aldettinger**
[ ] An issue should be filed for the change unless this is a trivial change (fixing a typo or similar). One issue should ideally be fixed by not more than one commit and the other way round, each commit should fix just one issue, without pulling in other changes.
[ ] Each commit in the pull request should have a meaningful and properly spelled subject line and body. Copying the title of the associated issue is typically enough. Please include the issue number in the commit message prefixed by #.
[ ] The pull request description should explain what the pull request does, how, and why. If the info is available in the associated issue or some other external document, a link is enough.
[ ] Phrases like Fix #<issueNumber> or Fixes #<issueNumber> will auto-close the named issue upon merging the pull request. Using them is typically a good idea.
[ ] Please run mvn process-resources -Pformat (and amend the changes if necessary) before sending the pull request.
[ ] Contributor guide is your good friend: https://camel.apache.org/camel-quarkus/latest/contributor-guide.html

### CODE REVIEW

File: extensions/json-validator/deployment/pom.xml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    </dependencyManagement>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.camel.quarkus</groupId>
+            <artifactId>camel-quarkus-core-deployment</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.camel.quarkus</groupId>
+            <artifactId>camel-quarkus-direct-deployment</artifactId>
```
Comment (jamesnetherton) [inline extensions/json-validator/deployment/pom.xml:53]: Do we really need to pull in the `direct` extension? We just need it to the itest module, right?

File: extensions/json-validator/runtime/pom.xml [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.camel.quarkus</groupId>
+            <artifactId>camel-quarkus-core</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.camel.quarkus</groupId>
+            <artifactId>camel-quarkus-direct</artifactId>
+        </dependency>
```
Comment (ppalaga) [inline extensions/json-validator/runtime/pom.xml:59]:
```suggestion
```

Same as above

File: integration-tests/json-validator/pom.xml [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.camel.quarkus</groupId>
+            <artifactId>camel-quarkus-json-validator</artifactId>
+        </dependency>
```
Comment (ppalaga) [inline integration-tests/json-validator/pom.xml:61]:
```suggestion
        </dependency>
        <dependency>
            <groupId>org.apache.camel.quarkus</groupId>
            <artifactId>camel-quarkus-direct</artifactId>
        </dependency>        
```

Here is where the direct extension belongs. IMO.

File: extensions/json-validator/deployment/pom.xml (hunk 1 above)
Comment (aldettinger) [reply extensions/json-validator/deployment/pom.xml:53]: Nice catch it makes sense.

**Pull request merged**